code/modules/games/cards.dm

```diff
@@ -127,8 +127,8 @@
 /obj/item/deck/examine(mob/user)
 	. = ..()
 	if(cards.len)
-		to_chat(user, "<br>There is still <b>[cards.len] card[cards.len > 1? "s" : ""]</b>.")
-	to_chat(user, SPAN_NOTICE("You can deal cards at a table with clicking at it with grab intent."))
+		to_chat(user, "<br>There [cards.len == 1 ? "is" : "are"]still <b>[cards.len] card\s</b>.")
```

Comment:
We should probably macro this at some point, there's a few places that could use it.